native/packages/react-native-bpk-component-button-link/src/BpkButtonLink-test.common.js

```diff
@@ -16,16 +16,20 @@
  * limitations under the License.
  */
 
+/* @flow */
```

Comment:
Are flow typing the tests as well?

Reply:
Did we make a decision not to do this? 🤔 😂

Reply:
I thought it was public API only, I might have missed something 😅